src/main/resources/templates/fragments/inputs/number.html

```diff
@@ -17,19 +17,22 @@
        th:if="${hasHelpText}"
        th:id="${inputName + '-help-text'}"
        th:text="${helpText}"></p>
-    <input type="text" inputmode="numeric"
-           class="text-input form-width--med"
-           th:id="${inputName}"
-           th:name="${inputName}"
-           th:placeholder="${placeholder}"
-           th:attr="
-            title=${hasTitle ? title : #messages.msg('general.inputs.number')},
-            pattern=${hasPattern ? pattern : '^\d*(\.\d{0,2})?$'},
-            aria-describedby=${hasHelpText ? inputName + '-help-text' : ''},
-            aria-labelledby=${hasAriaLabel ? ariaLabel : ''},
-            aria-invalid=${hasError}"
-           th:value="${fieldData.getOrDefault(inputName, '')}">
+    <div class="text-input-group form-width--med">
+      <input type="text" inputmode="numeric"
+             class="text-input form-width--med"
+             th:id="${inputName}"
+             th:name="${inputName}"
+             th:placeholder="${placeholder}"
+             th:attr="
+              title=${hasTitle ? title : #messages.msg('general.inputs.number')},
+              pattern=${hasPattern ? pattern : '^\d*(\.\d{0,2})?$'},
+              aria-describedby=${hasHelpText ? inputName + '-help-text' : ''},
+              aria-labelledby=${hasAriaLabel ? ariaLabel : ''},
+              aria-invalid=${hasError}"
+             th:value="${fieldData.getOrDefault(inputName, '')}">
+      <div th:if="${postfix != null}" th:text="${postfix}" class="text-input-group__postfix"></div>
```

Comment:
When using voiceover, the `postfix` text is not being described. Idk if this is a blocker - @spokenbird?


https://github.com/codeforamerica/form-flow/assets/9101728/35d6ec12-4787-40a1-93d9-61b23766ef80

Reply:
Nice catch! Thanks for thinking of this test.